### ISSUE TITLE
Unable to use an older tutorial due to missing function

### ISSUE THREAD

**Issue by Obsideaock** [tqdm/tqdm#1276]
I don't know if this is the correct place to ask this, but I don't know where else to so here it is,

I'm currently following a tutorial (https://towardsdatascience.com/ask-wikipedia-eli5-like-questions-using-long-form-question-answering-on-haystack-32cf1ca6c00e) and one of the code snippets is this

`from tqdm.notebook import tqdm
docs = []
batch_size = 10000
total_doc_count = 100000
with tqdm(total=total_doc_count) as progress_bar:
  for idx, article in enumerate(iter(wiki_snippets)):
    doc = convert_to_doc_dict(article)
    docs.append(doc)
    progress_bar.update(1)
    if idx % batch_size == 0:     
      document_store.write_documents(docs)      
      docs.clear()
    if idx >= total_doc_count:
      break  `

but in that snippet the function `convert_to_doc_dict` doesn't appear to exist anymore.
Is there a new function similar to this, or something else that can be used. Many thanks if anyone can help.

**Comment by chengs** [comment]
I think this is not the scope of `tqdm`. It does not contain the function `convert_to_doc_dict`
